src/components/AppSettingsDialog/AppSettingsDialog.vue

```diff
@@ -0,0 +1,260 @@
+<!--
+ - @copyright Copyright (c) 2020 Marco Ambrosini <marcoambrosini@pm.me>
+ -
+ - @author Marco Ambrosini <marcoambrosini@pm.me>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+<docs>
+Just nest the `AppSettingSections` component into `AppSettingsDialog`,
+providing the sectionTitle prop. You can put your settings within each
+`AppSettingsSection` component.
+
+```vue
+<template>
+	<div>
+		<button @click="settingsOpen = true">Show Settings</button>
+		<AppSettingsDialog :open.sync="settingsOpen" >
+			<AppSettingsSection sectionTitle="Example title 1">
+				Some example content
+			</AppSettingsSection>
+			<AppSettingsSection sectionTitle="Example title 2">
+				Some more content
+			</AppSettingsSection>
+		</AppSettingsDialog>
+	</div>
+</template>
+
+<script>
+export default {
+	data() {
+		return {
+			settingsOpen: false,
+		}
+	},
+}
+</script>
+```
+ </docs>
```

Comment:
```suggestion
<docs>
Just nest the `AppSettingSections` component into `AppSettingsDialog`,
providing the sectionTitle prop. You can put your settings within each
`AppSettingsSection` component.

```vue
<template>
	<div>
		<button @click="settingsOpen = true">Show Settings</button>
		<AppSettingsDialog :open.sync="settingsOpen" >
			<AppSettingsSection sectionTitle="Example title 1">
				Some example content
			</AppSettingsSection>
			<AppSettingsSection sectionTitle="Example title 2">
				Some more content
			</AppSettingsSection>
		</AppSettingsDialog>
	</div>
</template>

<script>
export default {
	data() {
		return {
			settingsOpen: false,
		}
	},
}
</script>
```
</docs>
```